app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -97,11 +126,12 @@ protected void onStop() {
         //Removing event listeners
         mRobot.removeOnRobotReadyListener(this);
         mRobot.removeOnCurrentPositionChangedListener(this);
+        mRobot.removeOnGoToLocationStatusChangedListener(this);
     }
 
     /**
      * Hide app's top bar when
-     * @param isReady is true
+     * @param "isReady" is true
```

Comment:
I think you're misunderstanding the block comments.

The description and parameters are not single sentence.

Here's an example:
```java
/**
 * This is a simple description of a method.
 * @param incomingDamage The amount of incoming damage
 * @return The amount of health hero has after attack
 */
public int successfullyAttacked(int incomingDamage) {
    // do things
    return 0;
}
```

Reply:
Is it better if it's written like this?
`    /**
     * Hide app's top bar when parameter "isReady" is true
     * @param isReady Robot status parameter
     */`

Reply:
@TheW00Xer Yes, looks good.